Lesson24/Ayse/create-post.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Create New Post</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>Create New Post</h1>
+
+      <form id="createPostForm" novalidate>
+        <div>
+          <label for="create-post-title">Title</label>
+          <input type="text" id="create-post-title" name="title" required />
+        </div>
+
+        <div>
+          <label for="create-post-body">Body</label>
+          <textarea id="create-post-body" name="body" required></textarea>
+        </div>
+
+        <button type="submit" class="button button--success">
+          Create Post
+        </button>
+
+        <button type="button" id="goHomeButton" class="button button-homePage">
+          Go to Home Page
+        </button>
+      </form>
+
+      <div id="create-success-message" class="success-message message"></div>
+
+      <div id="create-error-message" class="error-message message"></div>
+    </div>
+
+    <script src="./create-post.js"></script>
+
+    <script>
+      document
```

Comment:
There's no reason for adding JS code directly in HTML. Move it to the `create-post.js` file.

Reply:
The 'Go to Home Page' element don't actually need to be a button, you can make it <a> element then you don't need to add any JS code for it at all.